doc/admin/administration.md

```diff
@@ -124,21 +133,38 @@ boro-11
         Read Only: OK
         Volatile Memory Backup: OK
 ```
+### NVMe SSD Eviction and Hotplug
 
-- Manually Set Device State to FAULTY: `dmg storage set nvme-faulty`
+- Manually Evict an NVMe SSD: `dmg storage set nvme-faulty`
 
-Allows the admin to manually set the device state of the given device to FAULTY,
-which will trigger faulty device reaction (all targets on the SSD will be
-rebuilt and the SSD will remain in an OUT state until reintegration is
-supported).
+To manually evict an NVMe SSD (auto eviction will be supported in a future release),
+the device state needs to be set to "FAULTY" by running the following command:
 ```bash
 $ dmg -l boro-11 storage set nvme-faulty --uuid=5bd91603-d3c7-4fb7-9a71-76bc25690c19
 -------
 boro-11
 -------
   Devices
-    UUID:5bd91603-d3c7-4fb7-9a71-76bc25690c19 Targets:[0] Rank:1 State:FAULTY
+    UUID:5bd91603-d3c7-4fb7-9a71-76bc25690c19 Targets:[] Rank:1 State:FAULTY
+```
+The device state will transition from "NORMAL" to "FAULTY" (shown above), which will
+trigger the faulty device reaction (all targets on the SSD will be rebuilt and the SSD
+will remain evicted until device replacement occurs.
+
+- Replace an Evicted SSD with a New Device: `dmg storage replace`
+
+To replace an NVMe SSD with an evicted device and reintegrate it into use with
+DAOS, run the following command:
+```bash
+$ dmg -l boro-11 storage replace --old-dev=5bd91603-d3c7-4fb7-9a71-76bc25690c19 --new-dev=80c9f1be-84b9-4318-a1be-c416c96ca48b
```

Comment:
shouldn't the command/subcommand have some indication of bdev/nvme?

Reply:
yes that makes sense, similar to "dmg storage set nvme-faulty" I can update the command to "dmg storage replace nvme"?